website/sidebars.js

```diff
@@ -208,6 +208,7 @@ const sidebars = {
                     'getting-started/setup/instance-configuration/custom-domain/custom-ca-root-certificate',
                   ],
                 },
+                'getting-started/setup/instance-configuration/http-proxy',
```

Comment:
@sharat87 - Please confirm - The Proxy will be used in tandem with the custom domain configuration. Will it make sense to have this page added under the custom domain?

Reply:
CC: @danciaclara

Reply:
> The Proxy will be used in tandem with the custom domain configuration.

Not necessarily. Having or not having a custom domain isn't related to having to use a `HTTP_PROXY` or not. They're unrelated.